src/styles/main.scss

```diff
@@ -1,3 +1,7 @@
+@import url(./blocks/calendar.scss);
+
 body {
-  margin: 0;
+  margin: 40px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
```

Comment:
I believe main.scss should contain only imports, everything else we can move to another folder